src/software/ai/hl/stp/play/offense/offense_play_test.py

```diff
@@ -16,7 +17,8 @@ def test_offense_play(simulated_test_runner):
     # starting point must be Point
     ball_initial_pos = tbots_cpp.Point(-4.4, 2.9)
     # placement point must be Vector2 to work with game controller
-    tbots_cpp.Point(-3, -2)
+    tbots.Point(-3, -2)
+    field = tbots.Field.createSSLDivisionBField()
```

Comment:
`tbots` is undefined. You should be using `tbots_cpp`
```suggestion
    tbots_cpp.Point(-3, -2)
    field = tbots_cpp.Field.createSSLDivisionBField()
```
Once you made the change, you should verify that the test works as expected without errors by running: 
```
bazel run //software/ai/hl/stp/play/offense:offense_play_test -- --enable_thunderscope
```
You should try passing in `--aggregate` to test your aggregate validations, though I ran into some errors with this.

Reply:
Fixed! Sorry for the delay